src/init.cpp

```diff
@@ -726,6 +727,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     // Set this early so that parameter interactions go to console
     fPrintToConsole = GetBoolArg("-printtoconsole", false);
     fLogTimestamps = GetBoolArg("-logtimestamps", true);
+    fLogTimeMicros = GetBoolArg("-logtimemicros", false);
```

Comment:
Nit: Can you use the already present `fLogTimeMicros` which defaults to false instead of hardcoding it twice? (c.f. #6349)